src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -325,9 +341,212 @@ warning("PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrix
 return true;
     }
 
+  if(cache_size > 0)
+  {
+      cache_lm_file = true;
+      return cache_listmode_file();
+  }
+
    return false; 
 
-} 
+}
+
+template<typename TargetT>
+bool
+PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin<TargetT>::cache_listmode_file()
+{
+    if(!recompute_cache && cache_lm_file)
+    {
+        std::string curr_dir = FilePath::get_current_working_directory();
+        std::string cache_filename = "my_CACHE00.bin";
```

Comment:
will you still change this? If not, can you add it to the doxygen of this class?